kubernetes/deployment-staging.tmpl

```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cellect-staging-app
+  labels:
+    app: cellect-staging-app
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: cellect-staging-app
```

Comment:
Is this label going to conflict when (if?) we bring Cellect over to the azure k8s cluster?

Reply:
The other Cellect repo isn't deployed anywhere – this one is built on top of that one and just adds the Panoptes DB connection (unless I've misunderstood).

Reply:
that is correct - this repo includes the cellect-server gem https://github.com/zooniverse/cellect_panoptes/blob/c6057b99330affc226c6437c99215dc90b6f6b0f/Gemfile#L9

and as it's a rack app it's run via https://github.com/zooniverse/cellect_panoptes/blob/c6057b99330affc226c6437c99215dc90b6f6b0f/config.ru#L22